### PR TITLE
updating caption

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @vvaths
- [ ] @AnushikhaRai
- [ ] @BabagoudaPatil
- [ ] @NehaSoni-05
- [ ] @SharathKumar10
- [ ] @aayushi-arora
- [ ] @dmakshay
- [ ] @gouravpatel31
- [ ] @karunakarl
- [ ] @mbissa
- [ ] @moazzamkhan
- [ ] @mritunjay-rana
- [ ] @narendra-poc
- [ ] @pulipati2021
- [ ] @ssurbhi19
- [ ] @vyas000
- [ ] @webdeveloper4gupta

### PR DESCRIPTION
**Pull request by vvaths**
fixes #2

### CODE REVIEW

File: _slides/02-vvaths.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Awesome new caption!
```
Comment (vyas000) [inline _slides/02-vvaths.md]:
```suggestion
Amazing Magic Caption!!!!!
```

File: _slides/02-vvaths.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (vyas000) [inline _slides/02-vvaths.md:5]: good job! nice

**Review by vyas000 — CHANGES_REQUESTED** [review]
great. please make changes requested!

File: _slides/02-vvaths.md (hunk 2 above)
Comment (vvaths) [reply _slides/02-vvaths.md:5]: thanks!

**Review by vyas000 — APPROVED** [review]
ELGTM!

**Pull request merged**